src/main/java/study/step3/RacingGame.java

```diff
@@ -0,0 +1,32 @@
+package study.step3;
+
+import study.step3.domain.Car;
+import study.step3.move.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private final List<Car> carList = new ArrayList<>();
+    private final MoveStrategy moveStrategy;
+
+    public RacingGame(int carNum, MoveStrategy moveStrategy){
+        this.moveStrategy = moveStrategy;
+        for (int i = 0; i < carNum; i++) {
+            this.carList.add(new Car());
+        }
+    }
+
+    public List<Car> getCarList() {
+        return carList;
+    }
+
+    public void play() {
+        for (Car car : carList) {
+            if (moveStrategy.isSuccess()) {
+                car.move();
+            }
```

Comment:
```suggestion
            car.move(moveStrategy);
```

기존 방식과 Car에게 전략을 전달하는 방식 중 어떤 차이가 있을지 고민해 보시면 좋을 것 같아요